eval-bot/src/main/resources/sbot-defaults.cfg

```diff
@@ -1 +1,5 @@
-entityTimeout  = 30 seconds
+
+requiredMessagePrefix = ""
+
+http.entityTimeout    = 30 seconds
```

Comment:
I'm using Knobs instead of Typesafe config for this project. Regardless, I can rename the file to `reference.cfg`. And I could use `application.cfg` or some other name in the local dir.